test/new_relic/agent/javascript_instrumentor_test.rb

```diff
@@ -273,7 +273,7 @@ def test_data_for_js_agent_ignores_custom_attributes_by_config
     end
   end
 
-  def test_html_safe_if_needed_unsafed
+  def test_html_safe_if_needed_unsafe
```

Comment:
I'm not sure what this is trying to say (nor the next one)

Reply:
I think it's broken down like this: 
`test_`: required minitest prefix for all tests
`html_safe_if_needed`: method name https://github.com/newrelic/newrelic-ruby-agent/blob/ab2cf23ad6c4e9dadd7922871a1f8cf30088433f/lib/new_relic/agent/javascript_instrumentor.rb#L199-L202
`_unsafe`/`safe`:  way of breaking up the test cases into strings that are not html safe and strings that are html safe

Reply:
ok. so, it sounds like the `d` isn't needed/wanted. In which case I'm happy w/ this change.